src/libopenrave-core/jsonparser/jsonreader.cpp

```diff
@@ -344,7 +344,7 @@ class JSONReader
 
         // have to remove any duplicate names, prioritize ones that have higher index
         {
-            std::unordered_set<std::string> existingBodyNames;
+            boost::unordered_set<string_view> existingBodyNames;
```

Comment:
@rschlaikjer sorry that I missed this, but why don't we use `std::unordered_set<OpenRAVE::string_view>`?

cc @ntohge

Reply:
resolved by adding std::hash, https://github.com/rdiankov/openrave/pull/1412